rm_ros2_manual:
  ros__parameters:
    robot_type: "standard"
    chassis:
      accel_x:
        x: [ 50.0, 70.0, 90.0 ]
        y: [ 5.0, 7.0,9.0 ]
      accel_y:
        x: [ 50.0, 70.0, 90.0 ]
        y: [ 5.0, 7.0,9.0 ]
      accel_z:
        x: [ 50.0, 70.0, 90.0 ]
        y: [ 5.0, 7.0,9.0 ]
      topic: "/cmd_chassis"
      qos: 10.0
    vel:
      max_linear_x:
        x: [ 50.0, 70.0, 90.0 ]
        y: [ 5.0, 7.0,9.0 ]
      max_linear_y:
        x: [ 50.0, 70.0, 90.0 ]
        y: [ 5.0, 7.0,9.0 ]
      max_angular_z:
        x: [ 50.0, 70.0, 90.0 ]
        y: [ 5.0, 7.0,9.0 ]
      topic: "/cmd_vel"
      qos: 10.0
      power_limit_topic: "/cmd_chassis"
      gyro_move_reduction: 0.5
      gyro_rotate_reduction: 0.5
    gimbal:
      max_yaw_vel: 3.14
      max_pitch_vel: 3.14
      track_timeout: 0.1
      eject_sensitivity: 1.0
      gimbal_scale: 2.0
      topic: "/cmd_gimbal"
      qos: 10.0
    controllers:
      main_controllers:
        - gimbal_controller
